frontend/src/hooks/utilHooks.ts

```diff
@@ -41,12 +41,22 @@ export function useEditionAPIUrlTransformer(): (url: string) => string {
 export function useEditionApplicationPathTransformer(): (url: string) => string {
     const [editionUrl] = useEdition();
     const { data: edition } = useSWR(editionUrl, getEditionOnUrl);
-    return (url) => getQueryUrlFromParams(url, { edition: edition?.name });
+
+    return (url) => {
+        // make sure only a single edition is contained in the url
+        if (edition) {
+            if (url.includes("?edition=") || url.includes("&edition=")) {
+                // Edition already set -> remove
+                return url;
```

Comment:
hmmm.... that's not a remove? If i logout, what happens? If the path at logout is okey, this code can get rerolled

Reply:
The edition stays in the url on logout